lib/datadog/tracing/contrib/patcher.rb

```diff
@@ -55,6 +55,14 @@ def on_patch_error(e)
             Datadog.health_metrics.error_instrumentation_patch(1, tags: tags)
           end
 
+          def patch_results
```

Comment:
Actually, I think only the accessor should be exposed. See my other comment about this.